CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+## Expression Functions
+- Added `nonNull` for special evaluation handling for temporary null values. This may be useful in migrating code that is now producing many incompatbile argument warnings.
```

Comment:
I think it would be useful to explain a bit more about what `nonNull` does here. I assume it blocks the input until it's non-null, right?